gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.groovy

```diff
@@ -32,12 +33,19 @@ class BaselineErrorProne extends AbstractBaselinePlugin {
             errorprone "com.palantir.baseline:baseline-error-prone:${extractVersionString()}"
         }
 
-        project.tasks.withType(JavaCompile) {
-            options.compilerArgs += [
-                    "-XepDisableWarningsInGeneratedCode",
-                    "-Xep:EqualsHashCode:ERROR",
-                    "-Xep:EqualsIncompatibleType:ERROR",
-            ]
+        project.afterEvaluate { p ->
+            p.tasks.withType(JavaCompile) {
+                options.compilerArgs += [
+                        "-Xbootclasspath/p:${project.getConfigurations().getByName("errorprone").getAsPath()}",
```

Comment:
Does the -> arrow syntax make this any more lazy? We've used this in [conjure](https://github.com/palantir/conjure-java/blob/develop/conjure-java-verifier/build.gradle#L90) occasionally:

```groovy
 "-Xbootclasspath/p:${-> project.getConfigurations().getByName("errorprone").getAsPath()}",
```

Reply:
didn't make it work - I think the assignment forces evaluation